Rocket.Chat/Views/Cells/Chat/ChatMessageCell.swift

```diff
@@ -210,7 +210,7 @@ final class ChatMessageCell: UICollectionViewCell {
 extension ChatMessageCell: UIGestureRecognizerDelegate {
 
     func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRequireFailureOf otherGestureRecognizer: UIGestureRecognizer) -> Bool {
-        return false
+        return true
```

Comment:
@rafaelks I did not notice that if return `true` here the user would need to resign finger from the screen to fire event. I thought this will prevent other gesture to be fired.

I will revert this one.